controllers/operator_controller.go

```diff
@@ -203,31 +175,18 @@ func (r *OperatorReconciler) reconcile(ctx context.Context, op *operatorsv1alpha
 	// image we just looked up in the solution.
 	dep := r.generateExpectedBundleDeployment(*op, bundleImage)
 	if err := r.ensureBundleDeployment(ctx, dep); err != nil {
-		apimeta.SetStatusCondition(&op.Status.Conditions, metav1.Condition{
-			Type:               operatorsv1alpha1.TypeReady,
-			Status:             metav1.ConditionFalse,
-			Reason:             operatorsv1alpha1.ReasonInstallationFailed,
-			Message:            err.Error(),
-			ObservedGeneration: op.GetGeneration(),
-		})
+		// originally Reason: operatorsv1alpha1.ReasonInstallationFailed
```

Comment:
Are we going to replace this with some other conditions? Where are we tracking this?